test/com/google/enterprise/adaptor/documentum/DocumentumAdaptorTest.java

```diff
@@ -2626,20 +2628,22 @@ public void testStartPathDocContent() throws Exception {
 
   @Test
   public void testSlashInNameDocContent() throws Exception {
+    // Insert two documents with a "path" of START_PATH + "/aaa/bbb".
     // If the slash in the name is interpreted as a path separator,
     // we will try to crawl this document.
-    String folder = START_PATH + "/Slash";
-    insertFolder(getNowPlusMinutes(0), FOLDER.pad("FFF1"), folder);
+    String folder = START_PATH + "/aaa";
+    insertFolder(getNowPlusMinutes(0), FOLDER.pad("aaa"), folder);
     String mimeType = "text/plain";
-    String wrongName = "InName";
-    insertDocument(new Date(), DOCUMENT.pad("DDD0"), DOCUMENT.pad("DDD0"),
+    String wrongName = "bbb";
+    String wrongId = DOCUMENT.pad(wrongName);
+    insertDocument(new Date(), wrongId, wrongId,
         folder + "/" + wrongName, wrongName, mimeType, "Wrong Document");
 
     // The correct document, with an embedded slash in the file name.
-    String name = "Slash/InName";
+    String name = "aaa/bbb";
     String path = START_PATH + "/" + name;
     String content = "Right Document";
-    String id = DOCUMENT.pad("DDD1");
+    String id = DOCUMENT.pad("aaabbb");
```

Comment:
I don't think the relationship between the name and the ID matters, and indeed we are moving to a scheme where they are intrinsically related. However, I had thought about making this ID ccc, and am happy to do so.